apps/alpha-test-app/package.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "alpha-test-app",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "vue-tsc && vite build",
+    "preview": "vite preview"
+  },
+  "dependencies": {
+    "sit-onyx": "file:../../packages/sit-onyx",
```

fyi @larsrickert , when I only used `pnpm link ../../packages/sit-onyx`, the package.json was not modified so I worried that the next dev that wants to install will not get sit-onyx in here at all. that's why I added this line.
There as also alternatively that syntax
```suggestion
    "sit-onyx": "link:../../packages/sit-onyx",
```
but then I got that weird error in https://github.com/SchwarzIT/onyx/pull/157#discussion_r1458850251

I just hope this way it works fine. Could you try to get it running on your laptop if it works fine? :)

---

```suggestion
    "sit-onyx": "workspace:^",
```

Since we are in the monorepo we can use the `workspace` command from pnpm to link the package :)

---

thank youuu!